nvflare/apis/impl/study_manager.py

```diff
@@ -94,14 +126,18 @@ def list_active_studies(self) -> List[str]:
         Returns: list of study names
 
         """
-        now = datetime.datetime.utcnow()
+        today = date.today()
         active_studies = list()
-        for k, v in self._existing_studies:
-            if v["start_date"] >= now and v["end_date"] <= now:
-                active_studies.append(k)
+        for st in self._existing_studies:
+            meta = self._storage.get_meta(st)
```

Comment:
Will this get_meta raise Exception?
Should we handle that?

Reply:
Yes.  When it raises exception, that study should not be included.